lib/internal/process/warning.js

```diff
@@ -44,6 +127,6 @@ function setupProcessWarnings() {
       if (process.throwDeprecation)
         throw warning;
     }
-    process.nextTick(() => process.emit('warning', warning));
+    process.nextTick(doEmitWarning(warning));
```

Comment:
Since I was in here working, I took the opportunity to make a couple of additional cleanups that I'd been sitting on.